GetIntoTeachingApi/Controllers/OperationsController.cs

```diff
@@ -88,5 +91,18 @@ public void SimulateError()
 
             builder.Append("throw error");
         }
+
+        [HttpGet]
+        [Authorize]
+        [Route("trigger_location_sync")]
+        [SwaggerOperation(
+            Summary = "Manually triggers a location sync job",
+            OperationId = "TriggerLocationSync",
+            Tags = new[] { "Operations" })]
+        [ProducesResponseType(typeof(void), 200)]
+        public void TriggerLocationSync()
+        {
+            RecurringJob.Trigger(JobConfiguration.LocationSyncJobId);

```

Comment:
Can we add a test for this in `OperationsControllerTests` and it would also be worth testing that the `Authorize` attribute is present, similar to:

https://github.com/DFE-Digital/get-into-teaching-api/blob/master/GetIntoTeachingApiTests/Controllers/CandidatesControllerTests.cs#L29-L33